libs/sdk-ui-dashboard/src/model/store/permissions/permissionsSelectors.ts

```diff
@@ -41,44 +41,143 @@ export const selectCanManageWorkspace = createSelector(selectPermissions, (state
     return state?.canManageAnalyticalDashboard ?? false;
 });
 
 /**
- * Returns whether the current user has permissions to manage access rights.
+ * Returns whether the current user has permissions to add, remove, and list ACLs (Access Control Lists) on an object.
  *
- * @alpha
+ * @public
  */
 export const selectCanManageACL = createSelector(selectPermissions, (state) => {
     return state?.canManageACL ?? false;
 });
 
 /**
- * Returns whether the current user has permissions to create scheduled emails.
+ * Returns whether the current user has permissions to create a scheduled email object and a KPI alert object.
  *
- * @alpha
+ * @public
  */
 export const selectCanCreateScheduledMail = createSelector(selectPermissions, (state) => {
     return state?.canCreateScheduledMail ?? false;
 });
+
+/**
+ * Returns whether the current user has permissions to run MAQL DDL and DML, access a workspace staging directory.
+ *
+ * @public
+ */
+export const selectCanInitData = createSelector(selectPermissions, (state) => {
+    return state?.canInitData ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to upload CSV files via CSV Uploader.
+ *
+ * @public
+ */
+export const selectCanUploadNonProductionCSV = createSelector(selectPermissions, (state) => {
+    return state?.canUploadNonProductionCSV ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to download a complete report.
+ *
+ * @public
+ */
+export const selectCanExecuteRaw = createSelector(selectPermissions, (state) => {
+    return state?.canExecuteRaw ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to access workbench.
+ *
+ * @public
+ */
+export const selectCanAccessWorkbench = createSelector(selectPermissions, (state) => {
+    return state?.canAccessWorkbench ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to create a report object via API.
+ *
+ * @public
+ */
+export const selectCanCreateReport = createSelector(selectPermissions, (state) => {
+    return state?.canCreateReport ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to create a KPI object, KPI widget object, and an insight object via API.
+ *
+ * @public
+ */
+export const selectCanCreateVisualization = createSelector(selectPermissions, (state) => {
+    return state?.canCreateVisualization ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to modify and delete a metric, run MAQL DDL, run the MAQL validator, change metric visibility via the `unlisted` flag.
+ *
+ * @public
+ */
+export const selectCanManageMetric = createSelector(selectPermissions, (state) => {
+    return state?.canManageMetric ?? false;
+});
+
+/**
+ * Returns whether the current user has permissions to modify and delete a report object, change report visibility via the `unlisted` flag.
+ *
+ * @public
+ */
+export const selectCanManageReport = createSelector(selectPermissions, (state) => {
```

Comment:
This is PixelPerfect-specific it seems, so it is not usable for the Dashboard component imo.

Reply:
removed